spec/tictactoe_spec.rb

```diff
@@ -27,7 +31,15 @@ def present(board)
   end
 
   class FakePlayer
+    attr_reader :token
+
+    def initialize(moves:, token:)
+      @moves = moves
+      @token = token
```

Comment:
Do we need `token`? It doesn't look like the fake uses it.

Reply:
Hmmm....I believe we need it

Reply:
FWIW, initializers/constructors are not considered part of the public interface of an object. Those are free to vary across different concrete versions of an abstraction (the Player abstraction in our case). Technically, the `initialize` method is part of the _class_. We only need to ensure that the public API of the _object_ is consistent.

Given that, do we need the token?

Reply:
You mentioned that `token` is part of the public interface now. 👍